src/query-self.ts

```diff
@@ -1,4 +1,4 @@
-import { setMaxListeners } from 'events'
+import { setMaxListeners } from './utils.js'
```

Comment:
why? events should work in browser

Reply:
maybe it will with [this package](https://www.npmjs.com/package/events) in dependencies.

should i update the PR?

Reply:
yeah, lets try that instead

Reply:
i have updated. 

the `npm run test:node`runs tests ending with `.node.ts`, but `npm test` does not. is it the way it supposed to work?

Reply:
Are you sure, they are both same command?
```
    "test": "aegir test -t node",
    "test:node": "aegir test -t node",
```